src/index.html

```diff
@@ -7,13 +7,16 @@
   </head>
 
   <body>
-    <div class='orange'>
-    </div>
+    <div class='main-container'>
+      <div class='product-container'>
```

Comment:
there's no need to wrap the image inside a div, just add the class to the image and that's it